app/models/pipeline.rb

```diff
@@ -58,8 +58,6 @@ class Pipeline < ApplicationRecord # rubocop:todo Metrics/ClassLength
   delegate :item_limit, :batch_limit?, to: :workflow
 
   belongs_to :control_request_type, class_name: 'RequestType'
-  belongs_to :next_pipeline, class_name: 'Pipeline'
-  belongs_to :previous_pipeline, class_name: 'Pipeline'
 
```

Comment:
These are pretty ancient, and I don't think have been used the entire time I've been here. My code-archeology suggests that they used to be part of automated qc processes, with woudl advance requests through pipelines.